src/map/ai/controllers/mob_controller.cpp

```diff
@@ -1260,6 +1260,15 @@ bool CMobController::CanAggroTarget(CBattleEntity* PTarget)
         return false;
     }
 
+    // Don't aggro, I'm a normal CoP Fomor and you have low hate
+    if (PMob->m_Family == 115 && !(PMob->m_Type & MOBTYPE_NOTORIOUS) && (PMob->getZone() >= 24 && PMob->getZone() <= 28) && PTarget->objtype == TYPE_PC)
+    {
+        if (((CCharEntity*)PTarget)->getCharVar("FOMOR_HATE") < 8)
```

Comment:
nit: This wouldn't ever happen here, but technically we could have passed in a non player character here. Curious if we should be a little tighter with this casting. Thoughts?

Reply:
Checking for a valid object type before casting seems pretty normal to me.

Reply:
Oh, I missed the tail end of the if above this, apologies!